ocs_ci/utility/scripts/cherry_pick/cherry-picks.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -e
+
+upstream=origin
+branch_name=$( git rev-parse --abbrev-ref HEAD )
+commit=$( git rev-parse HEAD )
+github_remote=$( git remote -v |grep -m1 pbalogh |sed 's/.*:\(.*\)\/.*/\1/' )
```

Comment:
looks like the user "pbalogh" is hardcoded here

Reply:
Good catch, it was supposed to be replaced by $FORK ;)